apps/example/src/components/peaks/list/PeaksList.tsx

```diff
@@ -13,29 +13,39 @@ import {
   Spinner,
   Badge,
   DataGridHeadCell,
+  Stack,
 } from "@cloudoperators/juno-ui-components"
 
+import { Peak } from "../../../mocks/db"
+import useConfigStore from "../../../store/useConfigStore"
+
+import Mountain from "../../../assets/mountain.svg?react"
+
+import HelpTooltip from "../../common/HelpTooltip"
 import GenericCard from "../../common/GenericCard"
+import { Views, PeakFields, TooltipExplanation, DEFAULT_MEDIUM_APP_MARGIN } from "../../constants"
 import HintNotFound from "../../messages/HintNotFound"
-import { Views, PeakFields } from "../../constants"
 
-import { Peak } from "../../../mocks/db"
 import PeaksListItem from "./PeaksListItem"
-import HelpTooltip from "../../common/HelpTooltip"
-import { TooltipExplanation } from "../../constants"
+
+// Needs refactoring
```

Comment:
I really do not like this comments :( I am sorry,
1) They are not explicit what should be done
2) They should be TODO: marks usually